olive/olive_config.json

```diff
@@ -0,0 +1,243 @@
+{
+    "passes": {
+        "AppendPrePostProcessingOps": {
+            "module_path": "olive.passes.onnx.append_pre_post_processing_ops.AppendPrePostProcessingOps"
```

Comment:
the stringlized module full path is hard to maintain. User might forget to update the module path when it is changed.

How could we make sure the path here is updated whenever module path changes?

Reply:
As I understand, the change in path would require a PR and the pass would stop working if the string is not updated.

Reply:
> As I understand, the change in path would require a PR and the pass would stop working if the string is not updated.

That's correct.

Reply:
We can raise a specific error if this path couldn't find the pass.

Reply:
> Don't we have passes that can be loaded only in specific environment like Azure or OpenVino?

Could you help explain more?

The UT function here I mean is like to verify all of `module_path` in  `olive_config.json` are right to the olive build-in passes. 
In my rough thoughts, we can just read the `olive_config.json` and start a looper to check if the `module_path` is right. 
I think it is quite simple, mostly? 

The UT I mean is just  for our build-in passes. Otherwise, if any typos of the `module_path` are checked into main, we have to start a new PR then wait CI and approval.

Reply:
Done!